srcjs/shinyapp.js

```diff
@@ -310,6 +310,10 @@ var ShinyApp = function() {
       var show = condFunc(scope);
       var showing = el.css("display") !== "none";
       if (show !== showing) {
+        el.trigger({
+          type: 'shiny:conditional',
```

Comment:
This is useful in its own right, but it wasn't what I originally meant. I meant for a $(document).trigger() call at the top or bottom of $updateConditionals; this would let users create extensions that need to react any time any input or output changes. At least, I think that's what I was picturing? @daattali did you ask for something like this?